ui/src/message_popup/playtest/hmtca_group_review.jsx

```diff
@@ -0,0 +1,103 @@
+/* @flow weak */
+import React from 'react';
+import _ from 'lodash';
+
+import * as Api from '../../helpers/api.js';
+import Paper from 'material-ui/Paper';
+import {Card, CardTitle, CardText} from 'material-ui/Card';
+
+
+// For reviewing responses as a group.
+export default React.createClass({
+  displayName: 'HMTCAGroupReview',
+
+  propTypes: {
+    applesKey: React.PropTypes.string
+  },
+
+  getInitialState() {
+    return {
+      hasLoaded: false,
+      applesResponses: null
+    };
+  },
+
+  componentDidMount() {
+    this.refreshResponses();
+  },
+
+  refreshResponses() {
+    const {applesKey} = this.props;
+    Api.getApples(applesKey).end(this.onResponsesReceived);
```

Comment:
This loads the responses from the server based on: 1) the code they put in at the beginning, and 2) the bucket they chose.  So if they used the same code and same bucket and kept playing day-after-day, they'd keep seeing more and more responses in the review screen.